.devcontainer/devcontainer.json

```diff
@@ -2,21 +2,29 @@
 // README at: https://github.com/rocker-org/devcontainer-templates/tree/main/src/r-ver
 {
 	"name": "R (rocker/r-ver base)",
-	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
```

Comment:
@eeholmes what do you think about leaving comments in? My thought was there are some options we can't anticipate, so my thought was to leave in as many comments as possible to help people find what they need? Or is there a better way we could do this?